Frontend/src/components/AdminSolver.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+export default function AdminSolver({ solver}) {
+  return (
+    <div className="solver-view">
+        {solver.name}
+    </div>
+  )
+}
+
```

Comment:
Because the components should be doing different things. The Solver.js component is for the user, and has an onClick function which adds it to the interface of components that can be run. The admin should not have that onClick function, so I made a new component. Could I instead have had just one component and alter the handleOnClick() function in Solver.js to only fire when the logged in user is an end user? Yes. Yes I could.